hooks/useSkipFirstRender.tsx

```diff
@@ -0,0 +1,8 @@
+import { useEffect, useState } from "react";
+
+export function useSkipFirstRender() {
```

Comment:
It seems it's not being used anywhere

Reply:
Damn it! I stashed it, but that doesn't remove new files. Removing it now.